test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/xunit.runner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "http://json.schemastore.org/xunit.runner.schema",
+  "methodDisplay": "method",
```

Comment:
I wish. There are only two options. `"classAndMethod"` and `"method"`. The former is the default and is what results in the massively long test names in Test Explorer.